src/RwaSpell.sol

```diff
@@ -0,0 +1,204 @@
+pragma solidity 0.5.12;
+
+import "lib/dss-interfaces/src/dss/VatAbstract.sol";
+import "lib/dss-interfaces/src/dapp/DSPauseAbstract.sol";
+import "lib/dss-interfaces/src/dss/CatAbstract.sol";
+import "lib/dss-interfaces/src/dss/JugAbstract.sol";
+import "lib/dss-interfaces/src/dss/EndAbstract.sol";
+import "lib/dss-interfaces/src/dss/SpotAbstract.sol";
+import "lib/dss-interfaces/src/dss/GemJoinAbstract.sol";
+import "lib/dss-interfaces/src/dss/DaiJoinAbstract.sol";
+import "lib/dss-interfaces/src/dss/FlipAbstract.sol";
+import "lib/dss-interfaces/src/dapp/DSTokenAbstract.sol";
+import "lib/dss-interfaces/src/dss/IlkRegistryAbstract.sol";
+import "lib/dss-interfaces/src/dss/ChainlogAbstract.sol";
+import "lib/dss-interfaces/src/dapp/DSValueAbstract.sol";
+
+// NOTES:
+// renamed to RWA-001
+// flipmom / osm?
+// SF fees? just input dummy data for right now?
+
+interface RwaLiquidationLike {
+    function wards(address) external returns (uint256);
+    function rely(address) external;
+    function deny(address) external;
+    function init(bytes32, bytes32, address, uint48) external;
+    function tell(bytes32) external;
+    function cure(bytes32) external;
+    function cull(bytes32) external;
+    function good(bytes32) external view;
+}
+
+interface RwaConduitLike {
+    function wards(address) external returns (uint256);
+    function can(address) external returns (uint256);
+    function rely(address) external;
+    function deny(address) external;
+    function hope(address) external;
+    function nope(address) external;
+    function bud(address) external returns (uint256);
+    function kiss(address) external;
+    function diss(address) external;
+    function pick(address) external;
+    function push() external;
+}
+
+contract SpellAction {
+    // KOVAN ADDRESSES
+   //
+    // The contracts in this list should correspond to MCD core contracts, verify
+    // against the current release list at:
+    //     https://changelog.makerdao.com/releases/mainnet/1.1.4/contracts.json
+
+    ChainlogAbstract constant CHANGELOG = ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
+
+    address constant MCD_VAT         = 0xbA987bDB501d131f766fEe8180Da5d81b34b69d9;
+    address constant MCD_CAT         = 0xdDb5F7A3A5558b9a6a1f3382BD75E2268d1c6958;
+    address constant MCD_JUG         = 0xcbB7718c9F39d05aEEDE1c472ca8Bf804b2f1EaD;
+    address constant MCD_SPOT        = 0x3a042de6413eDB15F2784f2f97cC68C7E9750b2D;
+    address constant MCD_END         = 0x24728AcF2E2C403F5d2db4Df6834B8998e56aA5F;
+    address constant ILK_REGISTRY    = 0xedE45A0522CA19e979e217064629778d6Cc2d9Ea;
```

Comment:
Fetch these from the `CHANGELOG` instead of hard coding them using the new spell format.